backend/api.py

```diff
@@ -13,23 +13,32 @@
 logging.basicConfig(level=logging.INFO)
 
 class HintRequest(BaseModel):
-    name: str = Field(..., example="Two Sum")
     description: str = Field(..., example="Given an array of integers nums and an integer target, return indices of the two numbers such that they add up to target.")
     code: str = Field(..., example="class Solution:\n    def twoSum(self, nums: List[int], target: int) -> List[int]:\n        for i in range(len(nums)):\n            for j in range(i + 1, len(nums)):\n                if nums[i] + nums[j] == target:\n                    return [i, j]\n        return []")
 
 class HintResponse(BaseModel):
     hint: str
 
-hint_prompt_template = '''Problem:
-  {problem}
+hint_prompt_template = '''Problem Description:
+  {description}
 
   Code:
   {code}
 
-  I'm stuck on this LeetCode problem and don't know why my code isn't working. Can you give me a hint that is in the spirit of an interview? Only bring up one issue/optimization. Avoid providing explicit solutions but point out general areas of improvement or potential issues in the code.
+  I'm stuck on this LeetCode problem and don't know why my code isn't working. Whats wrong?'''
```

Comment:
Is this the new prompt we'll be using? Or just temporary to test out the LLM?

Reply:
Temporary so that we can see that it clearly identifies the error for testing purposes